examples/platform/k32w/app/project_include/OpenThreadConfig.h

```diff
@@ -44,6 +44,15 @@
 // Use smaller maximum interval to speed up reattaching.
 #define OPENTHREAD_CONFIG_MLE_ATTACH_BACKOFF_MAXIMUM_INTERVAL (60 * 10 * 1000) // default 1200000 ms
 
+// disable unused features
```

Comment:
Agree to remove these features as they are indeed unused within Matter - we would leave a lot of unused dead code in the Matter stack otherwise.
But are they not required for Thread stack certification ? Which triggers the question how the supported set of Thread features relates to Matter certification.

Reply:
To be honest, I don't know how Matter certification will work in conjunction with Thread certification. In my opinion. we shouldn't loose Thread certification if some features which are not used by Matter are disabled. Maybe someone experienced  with the certification process can comment.

Reply:
AFAIK at least Joiner functionality (from the below list) is now required to pass the official Thread certification for End Device / Router.